src/js/utils/__tests__/rxjsUtils-test.js

```diff
@@ -19,12 +19,29 @@ describe("linearBackoff", function() {
 
       // In test env we don't want to wait for the real wall clock
       // so we encode time intervals with a special helper `m.time`
-      const result = source.retryWhen(linearBackoff(2, m.time("--|")));
+      const result = source.retryWhen(linearBackoff(m.time("--|"), 2));
 
       m.expect(result).toBeObservable(expected);
     })
   );
 
+  it(
+    "retries infinitey when told to do so",
+    marbles(function(m) {
+      m.bind();
+
+      const source = m.cold("1--2#");
+      // const expected = m.cold("1--2--1--2----1--2------1--2");
+      // const timer = m.cold("1--2--1--2----1--2------1--2|");
+
+      // const result = source.retryWhen(linearBackoff(m.time("--|"), -1));
+      // m.expect(result.take(8)).toBeObservable(expected);
+      expect(() =>
+        source.retryWhen(linearBackoff(m.time("--|"), -1))
```

Comment:
not sure if this works as exptected, the commented out part above didnt (`take` is undefined on marbles `ColdObservable`) - will check if I can implement this without marbels.